llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -432,10 +432,10 @@ bool SPIRVInstructionSelector::spvSelect(Register ResVReg,
 
   case TargetOpcode::G_FMINNUM:
   case TargetOpcode::G_FMINIMUM:
-    return selectExtInst(ResVReg, ResType, I, CL::fmin, GL::FMin);
+    return selectExtInst(ResVReg, ResType, I, CL::fmin, GL::NMin);
```

Comment:
I'm a little suprised there were no other tests that needed to be updated than the ones I add.